linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/interceptor/impl/CommentInterceptor.scala

```diff
@@ -102,7 +103,15 @@ object ScalaCommentHelper extends CommentHelper {
   override val commentPattern: Regex = """^\s*//.+\s*""".r.unanchored
   private val scalaCommentPattern: String = "(?ms)([\"'](?:|[^'])*['\"])|//.*?$|/\\*.*?\\*/"
 
-  override def dealComment(code: String): String = code
+  override def dealComment(code: String): String = {
+    // Append code `val a=1` to prevent bugs that do not exit tasks for a long time
+    if (EntranceConfiguration.ENTRANCE_SCALA_APPEND_CODE_ENABLED.getValue) {
```

Comment:
Is it more reasonable to modify Scala Code Parser, go to append val linkisVar=1

Reply:
Yes, Clearer meaning